package-scripts.js

```diff
@@ -24,29 +33,34 @@ const tests = testTypes.map(cmd => {
   sh.pushd(base)
   let tests = _(sh.ls())
   .map(t => [t.split('.')[0], path.join(base, t)])
-  .map(([t, s]) => [t, process.env.CIRCLECI ? `MOCHA_FILE=reports/mocha-${t}.xml ${mocha} --reporter mocha-junit-reporter ${s}` : `${mocha} ${s}`])
+  .map(([t, s]) => {
+    const mochaString = process.env.CIRCLECI ? `MOCHA_FILE=reports/mocha-${t}.xml ${mocha} --reporter mocha-junit-reporter ${s}` : `${mocha} ${s}`
+    const concurretlyString = 'node node_modules/concurrently/dist/bin/concurrently.js --kill-others-on-fail --prefix-colors "dim" --prefix "[{name}]" --names "basic"'
+    return [t, `${concurretlyString} "${mochaString}"`]
+  })
   sh.popd()
   tests = process.env.TEST_SERIES === '1' ?
-    series(...tests.map(t => t[1]).value()) :
-    concurrent(tests.fromPairs().value())
+    tests.map(t => t[1].value()).join(' && ') :
+    tests.fromPairs().value()
   if (process.env.CIRCLECI) {
-    tests = series(mkdirp('reports'), tests)
+    tests = `${pkgManager} mkdirp reports && ${objectValuesToString(tests)}`
   }
   sh.config.silent = silent
-  return [cmd, series('nps build', tests)]
+  return [cmd, `${pkgManager} build && ${objectValuesToString(tests)}`]
 })
 
 module.exports = {
   scripts: {
     build: 'rm -rf lib && tsc',
     lint: {
-      default: concurrent.nps('lint.eslint', 'lint.tsc', 'lint.tslint'),
+      default: 'node node_modules/concurrently/dist/bin/concurrently.js --kill-others-on-fail --prefix-colors "dim,dim,dim" --prefix "[{name}]" --names "lint.eslint, lint.tsc, lint.tslint" \'nps lint.eslint\' \'nps lint.tsc\' \'nps lint.tslint\'',
       eslint: script('eslint .', 'lint js files'),
       tsc: script('tsc --noEmit', 'syntax check with tsc'),
       tslint: script('tslint -p .', 'lint ts files'),
     },
     test: Object.assign({
-      default: series.nps(...testTypes.map(t => `test.${t}`)),
+      default: testTypes.map(t => `test.${t}`).join(' && '),
     }, _.fromPairs(tests)),
```

Comment:
Do we even need this at all if we're not calling `nps test` anymomre?

Reply:
Yes, still using `nps` for some of the top level calls. Just removed nps-utils, which is carrying the `trim-newlines` modules.